Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -132,21 +132,29 @@ public virtual void OnInputClicked(InputClickedEventData eventData)
             // On each tap gesture, toggle whether the user is in placing mode.
             IsBeingPlaced = !IsBeingPlaced;
```

Comment:
Would we still want to toggle the "IsBeingPlaced" value in this case, or should we just do nothing if the AnchorStore isn't available yet?